packages/neos-ts-interfaces/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "@neos-project/neos-ts-interfaces",
+    "version": "1.4.1",
+    "description": "Neos domain-related TypeScript interfaces",
```

Comment:
Good Idea, we can even run test for the `typings` too :)

Reply:
Tests? How? What sort of tests?